validators/schain-upgrade.md

```diff
@@ -0,0 +1,164 @@
+
+```bash
+MONITORING_CONTAINERS=True
+DOCKER_LVMPY_STREAM=1.0.0
+MANAGER_CONTRACTS_ABI_URL=https://skale-se.sfo2.digitaloceanspaces.com/manager-abi-1.5.2.json
+IMA_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/rinkeby/IMA/1.0.0-develop.38/abi.json
+CONTAINER_CONFIGS_STREAM=1.1.1
+FILEBEAT_HOST=3.17.12.121:5000
+SGX_SERVER_URL=[By validator, setup SGX wallet first]
+DISK_MOUNTPOINT=[By validator, your attached storage /dev/sda or /dev/xvdd (this is an example. You just need to use your 2TB block volume mount point)]
+DB_PORT=[By validator]
+DB_ROOT_PASSWORD=[By validator]
+DB_PASSWORD=[By validator]
+DB_USER=[by validator]
+IMA_ENDPOINT=[by validator, GETH NODE ENDPOINT Rinkeby ]
+ENDPOINT=[by validator, GETH NODE ENDPOINT Rinkeby]
+```
+---
+
+#### Back up SGX after node set up 
+ [How to back up SGX](https://skale.network/docs/documentation/sgxwallet/docs/backup-procedure)
+
+### Step 2 - TestNet Phase 3.1 Versions for SKALE Chain creation
+
+#### Important Notes - Before Update
+
+This update includes changes to docker-compose option:
+
+with the update, validators would need to support docker-compose themselves: setup and update it if necessary. 
+
+Installing docker-compose during SKALE node init is now disabled.
+
+Consensus: the build introduces BLAKE3 hash and consensus DB changes (for better performance). Should not impact update process
+
+#### Important
+
+1. Validators need to update docker-compose version on their machines to 1.27.4, because we started using new docker-compose syntax (cpu_shares functionality in particular).
+2. Validators need to remove .skale/node_data/skale.db before skale node update , because we updated db schema and until we have the migration ready we need to fast through this step by removing the folder.
+3. Resource allocation file should be re-created on all nodes - this should be done automatically during node update :
+    Why resource allocation should be updated?
+        i. In this update, we’re adding CPU and memory limits for the IMA container (this will also affect sChain container allocation)
+        ii. We’re changing the approach to estimating available memory on the machine
+        iii. The resource allocation file generation procedure and structure were revised
+
+**Note:** If DISK_MOUNTPOINT was changed in .env it’s required to do skale resources-allocation generate before update.
+
+4. Validators' Ledger devices: make sure the contract_data is ALLOWED (this can be set to Not Allowed after Ledger software update)
+5. Due to Secure Enclave changes introduced for the SGX Wallet release candidate, validators will need a backup key for the update
+
+#### Use these Versions for set up
+
+**Validator CLI version**: 1.2.0-beta.1
+
+**Node CLI version**: 1.1.0-beta.3
+
+**SGX version: sgxwallet**: 1.58.7-develop.1
+
+**SKALE Manager version**: 1.6.0-develop.7
+
+**Skaled version**:  1.46-develop.63
+
+**Skale Admin version**: 0.17.0-beta.1
```

Comment:
Should be  1.1.0-develop.17

Reply:
@badrogger Can you please update the old versions?